Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
 
 members = [
-    "crd", "operator", "server"
+    "rust/crd", "rust/operator", "rust/operator-binary"
```

Comment:
As @djc  commented this can be replaced by `rust/*`

Reply:
See comment above, that won't work with our release script at the moment.